pkg/storaged/dialogx.jsx

```diff
@@ -312,7 +312,7 @@ export const SelectOne = (tag, title, options, choices) => {
 export const CheckBox = (tag, title, options) => {
     return {
         tag: tag,
-        title: "", // sic
+        title: options.row_title,
```

Comment:
Is that safe? That would often be `undefined` instead of `""`. Could be safe, it's just not obvious from seeing this patch.

Reply:
Oh, good catch.  Falling back to `""` explicitly is indeed much better.